### PR TITLE
Add Romanian community tournaments

### PR DESCRIPTION
**Pull request by Teetoos98**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/o!RT/2021/en.md [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+tags:
+  - o!RST 2021
+  - o!RT
+---
+
+# osu! Romania Summer Tournament 2021
+
+![{SHORT_TITLE} logo](img/logo.png)
+
+The **osu! Romania Summer Tournament 2021** (***o!RST 2021***) was a 1v1, double elimination, country-based osu! tournament for Romanian and Moldovan players hosted by ::{ flag=RO }:: [Nice Aesthetics](https://osu.ppy.sh/users/7782553), ::{ flag=RO }:: [FallenAngelD](https://osu.ppy.sh/users/5225626), and ::{ flag=RO }:: [Ahrinity](https://osu.ppy.sh/users/11587229). It was the 3rd instalment of the osu! Romania Tournament.
```
Comment (Nivalyx) [inline wiki/Tournaments/o!RT/2021/en.md:11]:
```suggestion
The **osu! Romania Summer Tournament 2021** (***o!RST 2021***) was a 1v1, double elimination osu! tournament for Romanian and Moldovan players hosted by ::{ flag=RO }:: [Nice Aesthetics](https://osu.ppy.sh/users/7782553), ::{ flag=RO }:: [FallenAngelD](https://osu.ppy.sh/users/5225626), and ::{ flag=RO }:: [Ahrinity](https://osu.ppy.sh/users/11587229). It was the 3rd instalment of the osu! Romania Tournament.
```

Comment (Nivalyx) [reply wiki/Tournaments/o!RT/2021/en.md:11]: (`country-based` implies that the tournament consists of countries facing up against each other as in the World Cup)

File: wiki/Tournaments/o!RT/2021/en.md [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+  2. [MYTK - Senkyou (Realazy) [Cubby's Insane]](https://osu.ppy.sh/beatmapsets/1262410#osu/2623949)
+- DoubleTime
+  1. [Sana - Byoumei Koiwazurai ([ Joey ]) [Insane]](https://osu.ppy.sh/beatmapsets/220636#osu/609741)
+  2. [DJ Fresh (feat. Rita Ora) - Hot Right Now (Radio Edit) (Asphyxia) [Kysphyxia's Insane]](https://osu.ppy.sh/beatmapsets/231114#osu/539090)
+
+## Match results
+
+### Grand Finals
+
+Saturday, 11 September 2021
```
Comment (Nivalyx) [inline wiki/Tournaments/o!RT/2021/en.md:269]:
```suggestion
Saturday, 11 September 2021:
```

File: wiki/Tournaments/o!RT/2021/en.md [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
+
+| Player 1 | | | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| \_AfterWind ::{ flag=RO }:: | 4 | **9** | ::{ flag=RO }:: **L9 ELOVATOR** | [#1](https://osu.ppy.sh/community/matches/90747076) |
+
+Sunday, 12 September 2021
```
Comment (Nivalyx) [inline wiki/Tournaments/o!RT/2021/en.md:275]:
```suggestion
Sunday, 12 September 2021:
```

Comment (Nivalyx) [reply wiki/Tournaments/o!RT/2021/en.md:275]: (and so on)

File: wiki/Tournaments/o!RT/2021/en.md [hunk 4]
```diff
@@ -0,0 +1,518 @@
...
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown "1st Place"](/wiki/shared/crown-gold.png "1st place") | 70€, 1x Lotus Keypad, Profile Badge |
+| ![Silver crown "2nd Place](/wiki/shared/crown-silver.png "2nd place") | 30€, 1x Lotus Keypad |
+
+![](img/badge.png "o!RST 2021 winner badge")
+
+**Note**: The tournament winner waived his Lotus Keypad prize, thusly one keypad was awarded to the 3rd place contestant instead.
```
Comment (Nivalyx) [inline wiki/Tournaments/o!RT/2021/en.md:35]:
```suggestion
**Note**: As the winner of the tournament waived his Lotus Keypad prize, the keypad was awarded to the 3rd place winner instead.
```

File: wiki/Tournaments/de.md [hunk 5]
```diff
@@ -251,6 +251,15 @@ Nicht offizielle Turniere/Wettkämpfe, die von den Communities veranstaltet werd
 | [osu! Indonesia Tournament #11 (2021)](OIT/2021) | 04.01.2021 | 28.03.2021 | ::{ flag=ID }:: [rho](https://osu.ppy.sh/users/1629553) | ::{ flag=ID }:: [Skydiver](https://osu.ppy.sh/users/4750008) | ::{ flag=ID }:: [Crezz](https://osu.ppy.sh/users/7108275) |
 | [osu! Indonesia Tournament #12 (2022)](OIT/2022) | 15.02.2022 | 03.04.2022 | ::{ flag=ID }:: [Lifeline](https://osu.ppy.sh/users/11367222) | ::{ flag=ID }:: [deeto](https://osu.ppy.sh/users/10069909) | ::{ flag=ID }:: [Thatnoobguy](https://osu.ppy.sh/users/11091594) |
 
+#### osu! Romania Tournament
+
+| Name | Start | Ende | ![Goldkrone][GCrown] | ![Silberkrone][SCrown] | ![Bronzekrone][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| osu! Romania Tournament 2018 (High Bracket) | 2018-04-17 | 2018-08-18 | ::{ flag=RO }:: [badeu](https://osu.ppy.sh/users/1473890/) | ::{ flag=RO }:: [Chamosiala](https://osu.ppy.sh/users/1469892) | ::{ flag=RO }:: [vali24dm](https://osu.ppy.sh/users/10398451/) |
```
Comment (Nivalyx) [inline wiki/Tournaments/de.md:258]:
```suggestion
| osu! Romania Tournament 2018 (High Bracket) | 17.04.2018 | 18.08.2018 | ::{ flag=RO }:: [badeu](https://osu.ppy.sh/users/1473890/) | ::{ flag=RO }:: [Chamosiala](https://osu.ppy.sh/users/1469892) | ::{ flag=RO }:: [vali24dm](https://osu.ppy.sh/users/10398451/) |
```

Comment (Nivalyx) [reply wiki/Tournaments/de.md:258]: (and so on; the Germans use a date format that's different from the common English date format)

image

File: wiki/Tournaments/fr.md [hunk 6]
```diff
@@ -251,6 +251,15 @@ Tournois et compétitions non-officiels gérés par la communauté.
 | [osu! Indonesia Tournament #11 (2021)](OIT/2021) | 04/01/2021 | 28/03/2021 | ::{ flag=ID }:: [rho](https://osu.ppy.sh/users/1629553) | ::{ flag=ID }:: [Skydiver](https://osu.ppy.sh/users/4750008) | ::{ flag=ID }:: [Crezz](https://osu.ppy.sh/users/7108275) |
 | [osu! Indonesia Tournament #12 (2022)](OIT/2022) | 15/02/2022 | 03/04/2022 | ::{ flag=ID }:: [Lifeline](https://osu.ppy.sh/users/11367222) | ::{ flag=ID }:: [deeto](https://osu.ppy.sh/users/10069909) | ::{ flag=ID }:: [Thatnoobguy](https://osu.ppy.sh/users/11091594) |
 
+#### osu! Romania Tournament
+
+| Nom | Début | Fin | ![Couronne en or][GCrown] | ![Couronne d'argent][SCrown] | ![Couronne de bronze][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| osu! Romania Tournament 2018 (High Bracket) | 2018-04-17 | 2018-08-18 | ::{ flag=RO }:: [badeu](https://osu.ppy.sh/users/1473890/) | ::{ flag=RO }:: [Chamosiala](https://osu.ppy.sh/users/1469892) | ::{ flag=RO }:: [vali24dm](https://osu.ppy.sh/users/10398451/) |
```
Comment (Nivalyx) [inline wiki/Tournaments/fr.md:258]:
```suggestion
| osu! Romania Tournament 2018 (High Bracket) | 17/04/2018 | 18/08/2018 | ::{ flag=RO }:: [badeu](https://osu.ppy.sh/users/1473890/) | ::{ flag=RO }:: [Chamosiala](https://osu.ppy.sh/users/1469892) | ::{ flag=RO }:: [vali24dm](https://osu.ppy.sh/users/10398451/) |
```

Comment (Nivalyx) [reply wiki/Tournaments/fr.md:258]: (and so on; the date formatting is also quite different in French)

image

**Review by TicClick — APPROVED** [review]
looks okay to me

**Pull request merged**